mpqp/execution/providers/google.py

```diff
@@ -237,8 +241,22 @@ def run_local_processor(job: Job) -> Result:
             f"Does not handle {job.job_type} for processor for the moment"
         )
     elif job.job_type == JobType.OBSERVABLE:
-        raise NotImplementedError(
-            f"Does not handle {job.job_type} for processor for the moment"
+        from cirq.ops.linear_combinations import PauliSum as Cirq_PauliSum
+        from cirq.ops.pauli_string import PauliString as Cirq_PauliString
+
+        assert isinstance(job.measure, ExpectationMeasure)
+
+        cirq_obs = job.measure.observable.to_other_language(
+            language=Language.CIRQ, circuit=cirq_circuit
+        )
+        assert type(cirq_obs) == Cirq_PauliSum or type(cirq_obs) == Cirq_PauliString
+
+        shots = 1000 if job.measure.shots == 0 else job.measure.shots
```

Comment:
Usually when we put `job.measure.shots` to 0, we want the exact expectation value, why here do we put manually shots to 1000?
If it is not possible to have the theoretical expectation value, then we should have return an error (DeviceJobIncompatibleError) earlier (in the check job device comptability for instance) or then implement it, right?

Reply:
done